deps/llvm.mk

```diff
@@ -6,12 +6,22 @@ LLVM_GIT_URL_COMPILER_RT ?= $(LLVM_GIT_URL_BASE)/compiler-rt.git
 LLVM_GIT_URL_LLDB ?= $(LLVM_GIT_URL_BASE)/lldb.git
 LLVM_GIT_URL_LIBCXX ?= $(LLVM_GIT_URL_BASE)/libcxx.git
 LLVM_GIT_URL_LIBCXXABI ?= $(LLVM_GIT_URL_BASE)/libcxxabi.git
+LLVM_GIT_URL_POLLY ?= $(LLVM_GIT_URL_BASE)/polly.git
 
 ifeq ($(BUILD_LLDB), 1)
 BUILD_LLVM_CLANG := 1
 # because it's a build requirement
 endif
 
+ifeq ($(USE_POLLY),1)
+BUILD_LLVM_CLANG := 1 # because it's a build requirement
+ifeq ($(USE_SYSTEM_LLVM),0)
+ifneq ($(LLVM_VER),svn)
+$(error USE_POLLY=1 requires LLVM_VER=svn)
```

Comment:
This is only because you haven't hooked up the tarball yet, right? I would hold off on that and make it a separate PR.

pulling from polly should probably be added to the update-llvm rule at the very end of this file though